mythril/laser/ethereum/transaction/transaction_models.py

```diff
@@ -52,7 +52,7 @@ def initial_global_state(self):
             self.gas_price,
             self.call_value,
             self.origin,
-            code=self.callee_account.code if self.code is None else self.code,
+            code=self.code or self.callee_account.code,
```

Comment:
perfect :1st_place_medal: . I forgot such thing existed.

Reply:
Creds to s0b0lev from discord for suggesting this. I didn't know either